site-src/geps/gep-917.md

```diff
@@ -126,23 +124,46 @@ Fields marked "Extended" support will eventually have conformance tests that
 lock in the behavior of that feature, and there will be a mechanism for implementations
 to tell the testing framework what extended fields they support.
 
-#### Testing framework
+## Testing framework
+
+Conformance tests will initially be structured as a CLI that runs on existing
+Kubernetes clusters. This may be expanded to include a testing library that can
+be imported by implementations. This tool will output test results in a
+structured format that can be used to generate reports. Where possible it will
+share code with the existing ingress-controller-conformance test suite.
+
+### Test Definitions
+
+Each set of tests will be built around a set of YAML manifests representing
+Gateway API resources. The tests will focus on how they expect a given
+controller to process those resources, configured via GatewayClass name. For
+example, tests may check to ensure that a controller correctly populates status,
+or that a request is appropriately routed to the desired backend with any
+relevant modifications.
+
+Each test definition will include the following:
+
+* Name
+* Description
+* Support Level
 
-The Gateway API project will provide a set of tests and harness to run them, such
-that an implementation may point the test harness at a GatewayClass or individual
-Gateway managed by that implementation and have the testing framework deliver a
-report on if it meets the conformance standard. The report must be
-machine-parsable.
+In the future, this will be expanded to include:
 
-There is a _lot_ of work to prepare this framework and introduce the initial
-round of tests, let alone to have complete test coverage. Having a minimal set
-of tests is a requirment for the API to graduate to `v1beta1` API stability level.
-(As per our upstream KEP). It's acceptable to begin with a small set and expand
-outward while the project is in beta, but having a full set of conformance tests
-that cover most of the API scope should be a requirement for declaring the API
-stable.
+* Docs URL
+* Function to determine feature enablement
 
-#### Certification process
+### Prepopulated Gateways
```

Comment:
FWIW I'm personally in favor of calling these "unmanaged" gateways as I feel that language is more clear at a glance.

Reply:
Although "unmanaged" makes sense for the most common use case here, I think "prepopulated" is a bit more broad here. For example, some implementations may have very slow provisioning, or some manual steps during initial provisioning followed by automated updates afterwards. What this is really referring to is the ability for implementations to prepopulate Gateways, for any reason, when running conformance tests.